CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Container file gids are now updated properly during syncuser. #840
 - Fix build for API.
 - Bindd mounts for `wwctl exec` are now on the same fs
+- Fix automatic build of overlays
```

Comment:
Should indicate the context in which automatic build was failing (e.g., when fetching an overlay with warewulfd).

Reply:
Its always hard to put something sensible in the CHANGELOG if only a refactoring is happerning